packages/fast-usdc/src/type-guards.js

```diff
@@ -26,3 +29,50 @@ export const makeProposalShapes = ({ PoolShares, USDC }) => {
   });
   return harden({ deposit, withdraw });
 };
+
+/** @type {TypedPattern<string>} */
+export const EvmHashShape = M.string({
+  stringLengthLimit: 66,
```

Comment:
interesting... We don't have patterns for "starts with xyz" and the like, so I had only thought of string length limits as denial-of-service protection.